R/check_folder.R

```diff
@@ -0,0 +1,246 @@
+#' Check the folder structure
+#'
+#' For the time being, this function only checks projects.
+#' Keep in mind that R packages have requirements for the folder structure.
+#' `check_cran()` checks those requirements.
```

Comment:
`?check_cran` gives a different explanation on what it does.  ;-)
```suggestion
#' `check_folder()` checks those requirements.
```